### ISSUE TITLE
Postinstall script is not correct

### ISSUE THREAD

**Issue by jjshinobi** [freshollie/jest-dynalite#70]
Hi,

After installing jest-dynalite@3.4.3 my `yarn install` fails on this command in jest-dynalite:

```
error .../node_modules/jest-dynalite: Command failed.
Exit code: 1
Command: yarn build && husky install
Arguments: 
Directory: .../node_modules/jest-dynalite
Output:
yarn run v1.22.10
$ rm -rf dist && tsc -p tsconfig.build.json
```

tsconfig.build.json doesn't exist in `jest-dynalite` package.

Is it correct postinstall command? 
```
"postinstall": "yarn build && husky install"
```

**Comment by jjshinobi** [comment]
I can see that it is fixed in 3.4.4. Closing.

**Comment by freshollie** [comment]
Apologies, hope this wasn't too disruptive
